float64 object_confidence
bool model_detection